typescript-carousel/main.ts

```diff
@@ -0,0 +1,78 @@
+const imgRelativeUrls = [
+  './images/001.png',
+  './images/004.png',
+  './images/007.png',
+  './images/025.png',
+  './images/039.png',
+];
+const $indicators = document.querySelector('#indicators');
+if (!$indicators) throw new Error('$indicators not found!');
+const indicatorList = Array.from(
+  $indicators.querySelectorAll('i')
+) as HTMLElement[];
+const $currentImg = document.querySelector('#current-img') as HTMLImageElement;
+const $mainRow = document.querySelector('#main-row');
+
+let position = 1;
+
```

Comment:
If our arrays start at `0` it would be easier to start out position at `0` and end at `4` (the length of our array). This way we don't have to do any sort of number manipulation and can use the value directly.